optional/README.md

```diff
@@ -13,19 +13,20 @@ yarn install
 
 ## Deploy Staging/Production
 
-To deploy the project you need to create accessible remote for Netlify. The normal use is generate that into gitlab.com
-After this you need generate new project inside Netlify and connect with GitLab repository.
+
+To deploy the project you need to create remote access for Netlify. This is commonly done from gitlab.com.
```

Comment:
Actually, it's not done **from** gitlab.com. How about "using a gitlab.com repository."?